assets/css/src/components-plugin/_stats.scss

```diff
@@ -10,20 +10,24 @@
   }
 }
 
-.mailpoet-stats-info {
-  border-bottom: 1px solid $color-tertiary-light;
-}
-
 .mailpoet-stats-info,
 .mailpoet-stats-general {
   background-color: $color-white;
   margin: 0;
-  padding: 0 $grid-gap $grid-gap;
+  padding: $grid-gap;
+}
+
+.mailpoet-stats-info {
+  border-bottom: 1px solid $color-tertiary-light;
+  line-height: 24px;
```

Comment:
Maybe $heading-font-size-h3 ?

Reply:
I skipped using any variable here, as it doesn't really correlate with any other UI. It's there just so the lines in each column are aligned.